### ISSUE TITLE
Any values for turning off or save power for internal Radeon GPU

### ISSUE THREAD

**Issue by Anatoliy-Mey** [FlyGoat/RyzenAdj#160]
Hello, I've got notebook Ryzen 7 with external videocard RTX2060 connected to external monitor, and I don't need internal Radeon from Ryzen at all. So my question is: theres any way to trun it off or at list set very low value limits for power and temperature saving?

**Comment by Falcosc** [comment]
No, it is already at the lowest possible power state if not in use. You can turn the notebook screen of to try to get the integrated gpu into an even deeper sleep state.
